### PR TITLE
Blog post for iOS release 3.0.0

### PR DESCRIPTION
**Pull request by rafaelks**
## Progress

- [x] 3D Touch;
- [x] Quick actions;
- [x] Much better iPad UI;
- [x] Add commits, PRs and files changed stats;
- [x] GIFs

### CODE REVIEW

**Comment by rafaelks** [comment]
@isabellarussell Started this document so we can start discussing about this blog post, feel free to edit it or leave feedbacks please. 👍

File: _posts/2018/07/ios_3_0_0_released_app_store.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+(GIF SHOWING THE STATUS OF THE CONNECTION) 
+
+Also, if you were connected until you received couple of messages, if you loose your connectivity, you can still read the entire message when you enter in the room. We believe this change will provide a smoother experience for your on the daily basis.
+
+## Servers selection is more visible
+
+# Theming
+
+Yes!!! Now you can choose between light, dark and black themes. This project was developed by Samar Sunkaria during the GSoC project 2018. We choose this options of themes because we think it’s gonna look amazing on your iPhone and iPad, specially the black theme for the iPhone X users.
```
Comment (cardoso) [inline _posts/2018/07/ios_3_0_0_released_app_store.md]: Just make clear Samar will continue working on theme related features, and users and server admins can expect more customization options in the near future 😃

Comment (cardoso) [reply _posts/2018/07/ios_3_0_0_released_app_store.md]: Light, Dark and Black Themes is the first step

**Comment by isabellarussell** [comment]
@rafaelks please see my edits in the post. we are lacking the GIFS and screengrabs.

**Comment by rafaelks** [comment]
@thiagosanchz Let's write something about the new icon? :-)

**Comment by rafaelks** [comment]
@isabellarussell Finally added all the images and GIFs... can you please take a look?

**Comment by rafaelks** [comment]
@isabellarussell Awesome! Thanks a lot, will push the button with the release too. 👍

**Pull request merged**